Meadowlark-js/backends/meadowlark-opensearch-backend/test/setup/OpenSearchSetup.ts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { Config } from '@edfi/meadowlark-utilities';
+import path from 'path';
+import { DockerComposeEnvironment, StartedDockerComposeEnvironment, StartedTestContainer } from 'testcontainers';
+
+let environment: StartedDockerComposeEnvironment;
+const containerName = 'opensearch-test';
+
+export async function setupOpenSearch() {
+  let container: StartedTestContainer;
+  try {
+    const port = 8200;
```

Comment:
Just an observation, no action: we are running our default solution on port 9200, so this should be fine.